addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6618,6 +6618,42 @@ registry.ImageTools = ImageHandlerOption.extend({
     // Options
```

Comment:
Behaviour:
- We have to click on undo 3 times to undo the operation => it should only be one :+1: